tic-tac/script.js

```diff
@@ -83,4 +83,15 @@ function mainMenu(){
 
 function clean(dept){
 
+}
+
+function createDiv(elem, idParam){
+	var d = document.getElementById(elem);	
+	if(i%2==0){
```

Comment:
Please add the check for the existence of div, delete the old version of the script, and then test the system